### ISSUE TITLE
Leaderboard doesn't show google plus people (iOS)

### ISSUE THREAD

**Issue by skirners1** [playgameservices/play-games-plugin-for-unity#869]
Hello!

I just released my game (Fiskmåsen) on AppStore. The first person to test the game could see my developer account high score, until I logged in with that account on my iPad and played a game. Then my developer account disappeared from his leaderboards. I could never see my friends high score. The only person showing on one of my leaderboards (Flest Dödsfall), is a test player from the team on AppStore that approved the game, who probably never logged in with Google plus.

So I believe that there is a bug while getting the data from the GPGS-server, that makes players logged in through the game disappear some where along the way.

If you like you can try logging in to my game and see check the high score list 'Flest Dödsfall', where you will see how many times you died and how many times the tester died.

The game is yet to be approved by GPS, but I will come back with more intel once I have played it on android.

I read here on the forum that other players have similar problems with retrieving players from the leaderboards.

I'm a beginner on making games. Sorry if I cause any inconveniences. 
Im using version 0.9.26.

Best regards,
Lucas Ivarsson

**Comment by skirners1** [comment]
Hello!

I have now tested the game on android. It seems that any account associated with iOS devices logged in aren't available either on iOS or android when viewing the leaderboards. :-(

Is there any plans for fixing this any time soon?

**Comment by claywilkinson** [comment]
Thanks for the feedback. At this time there are no immediate plans for addressing this issue.  If that changes, there will be plenty of communication about it here and other locations.

**Comment by skirners1** [comment]
Hello Clay!

The players are authenticated on android, and the same code is used on iOS, so I think that they are authenticated. Maybe not. Maybe the authentication is what screw things up? I used an account on my android device first and it worked. when I used the same account on iOS it didn’t work, and stopped working back on android. Maybe there is a column in a database somewhere on the google server that is supposed to be empty, as in not using iOS, for the framework to function, or for a json to be properly disassembled?

I would really like to help out. However I’m such a super novice at programming. Is there any guide I can follow, to how I can produce such a log you mention?


Best regards,
Lucas Ivarsson
